src/DataDepsGenerators.jl

```diff
@@ -46,41 +80,51 @@ include("Figshare.jl")
 include("JSONLD/JSONLD.jl")
 
 function message(meta)
-    escape_multiline_string("""
-    Dataset: $(meta.fullname)
-    Website: $(meta.website)
-    $(meta.description)
-    """) |> strip
+    netString =  """
+    register(DataDep(
+        \"$(meta.shortname)\",
+        \"\"\""""
+    netString = format_meta(meta.website, netString, "Website")
+    netString = format_meta(format_authors(meta.author), netString, "Author", true)
+    netString = format_meta(meta.maintainer, netString, "Maintainer", true)
+    netString = format_meta(meta.license, netString, "License", true)
+    netString = format_meta(meta.publishedDate, netString, "Date of Publication", true)
+    netString = format_meta(meta.createDate, netString, "Date of Creation", true)
+    netString = format_meta(meta.modifiedDate, netString, "Date of Modification", true)
+    netString = format_meta(meta.description, netString, missing,  true)
```

Comment:
add a `netString *= "\n"`
before `format_meta(meta.description)

to leave a gap between the "Front matter" of mostly short fields,
and the description/citation details